### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by danieljgroso**
adding a general pull request template for the GI_mapping pipeline in a hidden dir, /.github

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```
Comment (cansavvy) [inline .github/pull_request_template.md]:
```suggestion
<!-- Please delete options that are not relevant. --> 
```
Optional little thing, you can use this type of html comment notation in order to make the instructions to the person who is filling out the template appear to them but not in the final rendered markdown.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
```
Comment (cansavvy) [inline .github/pull_request_template.md]: This looks like you may have borrowed this from somewhere on the internet? Is that right? If so, can you leave an html comment at the top that has the URL from where you borrowed this?

**Review by cansavvy — COMMENTED** [review]
I think this is a good start, @danieljgroso ! As we work on the project we can keep in mind how we might want to alter this pull request template to better suit this project's evolving needs.

File: .github/pull_request_template.md (hunk 2 above)
Comment (danieljgroso) [reply .github/pull_request_template.md]: done!

**Pull request merged**